pkg/metrics/metrics.go

```diff
@@ -0,0 +1,98 @@
+package metrics
+
+import (
+	"net/http"
+	"strings"
+
+	"github.com/prometheus/client_golang/prometheus"
+	"github.com/prometheus/client_golang/prometheus/promhttp"
+)
+
+// Error to be  returned when marketplace is unable to reach quay.io.
+type OpsrcError string
+
+const (
+	// Default values which are to be used as labels
+	// for prometheus metrics.
+	Error     = "opsrc_error"
+	OpsrcName = "opsrc_name"
+
+	// Path, Port and Host where custom metrics would be exposed.
+	MetricsPath = "/metrics"
+	MetricsPort = "8383"
+	MetricsHost = "0.0.0.0"
+
+	// Error message to be returned when quay.io is down.
+	UnreachableError OpsrcError = "quay_unavailabe"
```

Comment:
Documentation for Global variables should start with the name of the global variable. This applies in all places. Please see marketplace code for examples.